### ISSUE TITLE
Items on shelves are not transferrable via the "Transfer Menu"

### ISSUE THREAD

**Issue by spi3** [RimworldTogether/Rimworld-Together#163]
### Description

In both the global market menu as well as the caravan/settlement transfer menu, items which are on shelves are not available.

The workaround is to prioritize the item into a ground stockpile and then initiate the transfer.

### OS

Windows

### Reproduction steps

```bash
1. Open the world map
2. Click on "Global Market Menu"
3. Search for an item which is on a shelf
4. The shelved item will not be visible in the "Transfer Menu"

Alternative method:
1. Caravan to a player settlement
2. Initiate transfer
3. Player receiving the transfer can not select items which are on shelves.
```

### Client logs

```bash
N/A
```

### Server logs

```bash
N/A
```

**Comment by Byte-Nova** [comment]
This issue has been already fixed at [#152](https://github.com/RimworldTogether/Rimworld-Together/pull/152). Will be available at next release, thanks for pointing it out anyways :fire:

**Comment by spi3** [comment]
Oh awesome. Thanks for the follow up!
